src/PageLayout/PageLayout.tsx

```diff
@@ -397,6 +397,7 @@ export type PageLayoutContentProps = {
   width?: keyof typeof contentWidths
   padding?: keyof typeof SPACING_MAP
   hidden?: boolean | ResponsiveValue<boolean>
+  role?: string
```

Comment:
I'm wondering, is this to keep backwards compatibility with existing implementations out there? We could go all in on making the developer implicitly add the `<main>` within `PageLayout.Content` and not allow usage of `role`, but I'm curious on your thoughts here.